app/src/main/index.js

```diff
@@ -10,11 +10,12 @@ let semver = require('semver')
 let event = require('event-to-promise')
 let toml = require('toml')
 let axios = require('axios')
+let Raven = require('raven')
 
 let pkg = require('../../../package.json')
 let relayServer = require('./relayServer.js')
 let addMenu = require('./menu.js')
-let config = require('../../../config.js')
+let settings = require('../../../config.js')
```

Comment:
requiring config.js and naming as settings?

Reply:
because of a conflict with https://github.com/cosmos/voyager/blob/c37327e14f41732d6c2d15016fdf58febaba6bec/app/src/main/index.js#L478
I will rename the other instance then.